### PR TITLE
Single request for hpp

### PR DESCRIPTION
**Pull request by akhilamohanan**

### CODE REVIEW

File: pkg/controller/network_policy.go [hunk 1]
```diff
@@ -1060,6 +1081,99 @@ func (cont *AciController) buildServiceAugment(subj apicapi.ApicObject,
 	}
 }
 
+func (cont *AciController) handleMulNetPolUpdate(nps []interface{}) bool {
```
Comment (shastrinator) [inline pkg/controller/network_policy.go:1084]: Can we take a bool argument to call WriteMulApicObjects and use the existing handleNetPolUpdate function?

File: pkg/apicapi/apic_sync.go [hunk 2]
```diff
@@ -137,6 +137,113 @@ func (conn *ApicConnection) apicObjCmp(current ApicObject,
...
+		return
+	}
+	for _, val := range *deleteBody {
+		dn, okdn := val.Attributes["dn"]
+		if okdn && dn == deldn {
+			attrs := make(map[string]interface{})
+			attrs["dn"] = val.Attributes["dn"]
+			attrs["status"] = "deleted"
+			val.Attributes = attrs
+			conn.log.Debug("deleteBody: ", *deleteBody)
```
Comment (shastrinator) [inline pkg/apicapi/apic_sync.go:151]: Assuming all the logs will be cleaned up with another pass

File: pkg/apicapi/apic_metadata.go [hunk 3]
```diff
@@ -154,8 +154,7 @@ func injectedSvcPortNormalizer(b *ApicObjectBody) {
 var metadata = map[string]*apicMeta{
 	"fvTenant": {
 		attributes: map[string]interface{}{
-			"name":      "",
-			"nameAlias": "",
+			"name": "",
```
Comment (shastrinator) [inline pkg/apicapi/apic_metadata.go:157]: Was this change necessary?

File: pkg/controller/network_policy.go (hunk 1 above)
Comment (akhilamohanan) [reply pkg/controller/network_policy.go:1084]: Existing handleNetPolUpdate is handling one np at a time. Here we need multiple nps at a time

File: pkg/apicapi/apic_sync.go (hunk 2 above)
Comment (akhilamohanan) [reply pkg/apicapi/apic_sync.go:151]: Yes. I will remove all the debug logs that is not necessary.

File: pkg/apicapi/apic_metadata.go (hunk 3 above)
Comment (akhilamohanan) [reply pkg/apicapi/apic_metadata.go:157]: Yes. Since we are sending POST request to <tenant>.json and in the request body, if nameAlias is empty it was replacing the tenant alias (openupi) in our case to empty string

File: pkg/controller/network_policy.go (hunk 1 above)
Comment (shastrinator) [reply pkg/controller/network_policy.go:1084]: Each inner loop is the handleNetPolUpdate function. Can we not call in a loop and omit the writing to Apic part with a flag?

Comment (akhilamohanan) [reply pkg/controller/network_policy.go]: Moved common code to a separate function.

**Review by shastrinator — APPROVED** [review]
LGTM

**Comment by akhilamohanan** [comment]
Handled in https://github.com/noironetworks/aci-containers/pull/1015
